express-get/server.ts

```diff
@@ -0,0 +1,65 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+    select
+        "title"
+    from
+        "films"
+    order by "replacementCost" desc
+    limit 2;
+    `;
+    const result = await db.query(sql);
+    const films = result.rows;
+    res.json(films);
+  } catch (err) {
+    next(err);
+  }
+});
+
+app.get('/api/films/:filmId', async (req, res, next) => {
+  try {
+    const { filmId } = req.params;
+    if (filmId === undefined) {
```

Comment:
This check is fine, but it's not required since express will never route to this path if there is no filmId. So whenever this code executes, you are guaranteed that filmId exists.

Also, the next check (`isNumber`) will catch the case where `filmId` does not exist, should that ever happen for some reason.